lldb/packages/Python/lldbsuite/test/lldbtest.py

```diff
@@ -751,6 +751,8 @@ def setUpCommands(cls):
             "settings set symbols.enable-external-lookup false",
             # Inherit the TCC permissions from the inferior's parent.
             "settings set target.inherit-tcc true",
+            # Based on https://discourse.llvm.org/t/running-lldb-in-a-container/76801/4
+            "settings set target.disable-aslr false",
```

Comment:
The reason for change is in the Discourse link. This probably needs to be done in a separate commit.